internal/venice-common/src/main/proto/VeniceReadService.proto

```diff
@@ -0,0 +1,27 @@
+syntax = 'proto3';
+package com.linkedin.venice.protocols;
+
+option java_multiple_files = true;
+
+service VeniceReadService {
+  rpc get (VeniceClientRequest) returns (VeniceServerResponse) {}
+  rpc batchGet(VeniceClientRequest) returns (VeniceServerResponse) {}
+}
+
+message VeniceClientRequest {
+  uint32 partition = 1;
+  string keyString = 2;           // used for single get
+  bytes  keyBytes = 3;            // used for batch get
+  sint32 requestTimeoutInNS = 4;
+  string resourceName = 5;
+  string storeName = 6;
+  bool   isStreamingRequest = 7;
+  bool   isRetryRequest = 8;
+}
+
+message VeniceServerResponse {
+  sint32 schemaId = 1;
+  bytes  data = 2;
+  uint32 dataSize = 3;
+  bytes  serializedArr = 4;
```

Comment:
Is `dataSize` necessary since `data` is already a byte array?

Also what is `serializedArr`?
I suggest we add enough documents in the proto definition, and I assume they will be carried over to the generated java classes, right?

Reply:
it is not necessary anymore. I had utilized those fields at a previous iteration while testing and understanding how responses are sent back to the client. We no longer need those fields.